src/models/llama/modelling_llama_skip.py

```diff
@@ -52,15 +52,16 @@ def _init_components(self, config, layer_idx):
         self.input_layernorm = LlamaSkipRMSNorm(config.hidden_size, eps=config.rms_norm_eps)
         self.post_attention_layernorm = LlamaSkipRMSNorm(config.hidden_size, eps=config.rms_norm_eps)
 
-    def _set_mlp_train(self, config):
+    def _set_mlp_train(self, config, layer_idx):
```

Comment:
lets make this optional = none instead in skip mlp class?

Reply:
Sure, I can add that.